sumologic/resource_sumologic_slo.go

```diff
@@ -8,15 +8,13 @@ import (
 	"regexp"
 )
 
-const sloAggregationRegexString = `^(Avg|Min|Max|Sum|(p[5-9][0-9])(\.\d{1,3})?$)$` // TODO update it to allow lower pct values
-const sloAggregationWindowRegexString = `^[0-9]{1,2}(m|h)$`                        // TODO make it exact of min 1m and max 1h
+const sloAggregationWindowRegexString = `^[0-9]{1,2}(m|h)$` // TODO make it exact of min 1m and max 1h
```

Comment:
why are we keeping this validation ?

Reply:
hmm, we can remove this. It is being done in backend already, right?

Reply:
Yep